src/cdomains/floatDomain.ml

```diff
@@ -71,11 +82,27 @@ module FloatInterval = struct
         Some (l'', h'')
     | _, _ -> None
 
-  let hash = Hashtbl.hash
+  let maximal = function
+    | Some (_, h) -> Some h
+    | _ -> None
 
-  let show = function
-    | None -> "Float[Top]"
-    | Some (low, high) -> "Float [" ^ string_of_float low ^ "," ^ string_of_float high ^ "]"
+  let minimal = function
+    | Some (l, _) -> Some l
+    | _ -> None
+
+  let to_float = function
+    | Some (l, h) when l = h -> Some l
+    | _ -> None
+
+  let maximal = function
+    | Some (_, h) -> Some h
+    | _ -> None
+
+  let minimal = function
```

Comment:
two times the same implementation of `minimal` and `maximal`, one is unused (line 101 and 89; line 97 and 85)